### ISSUE TITLE
Error: EXCEPTION_ACCESS_VIOLATION when rendering using Network Render Addon

### ISSUE THREAD

**Issue by byrnes** [rlguy/Blender-FLIP-Fluids#328]
When rendering, I often get the Error: EXCEPTION_ACCESS_VIOLATION, and blender quits completely. This happens every 3 or 4 frames when rendering, on nearly any file I try to render. This happens when using fullscreen as well as the image editor view, and also happens when using the network render addon.

**Comment by rlguy** [comment]
Hey ConnorB123,

Do you mean that [this workaround](https://github.com/rlguy/Blender-FLIP-Fluids/issues/323#issuecomment-393563990) is not preventing crashes for you?

- Ryan

**Comment by ConnorB123** [comment]
Yes, that workaround does not work for me. Sometimes my renders will not crash, but most times they do.

**Comment by rlguy** [comment]
Hey ConnorB123,

Thanks for reporting this! This is actually the first report of the workaround not solving render crashes so I would like to get some more information so this can be solved for you and anyone else who may be experiencing the issue. I have a few questions that would help me learn more about this:

1. **What is your system information?**
    - Operating System
     - Blender System Information (attach file: **Blender > Help > Save System Info**)
    - Addon Version
    - CPU Model
    - GPU Model
2. **Are you using an official release version of Blender? Or an alternative build such as a nightly build or custom branch?**
3. **Are you using the the denoising feature?** - The denoising feature can be unstable and cause EXCEPTION_ACCESS_VIOLATION errors with certain setting configurations.
4. **Do you notice any patterns that relate to the frequency of crashes?** Examples:
    - Increase when rendering whitewater
    - Decrease after a system restart
    - GPU rendering vs CPU rendering
    - High resolution vs low resolution simulations
    - Low samples vs high samples
5. **Do the crashes occur at the same stage of rendering?**
    - During render initialization (right before rendering)?
    - During rendering?
    - After rendering completes?
6. **Do the crashes occur when [rendering from the command line](https://docs.blender.org/manual/en/dev/render/workflows/command_line.html#animation)?** Command line rendering seems to be the most stable method of rendering

Command:
```
/path/to/blender.exe -b /path/to/blend_file.blend -a
```

For example, my command looks like this (using Windows CMD):
```
"C:\Program Files\Blender Foundation\Blender\blender.exe" -b "C:\Users\ryanl\Documents\Blender\my_blender_scene.blend" -a
```

Thanks for providing any additional information, and sorry for the inconvenience! Hopefully I'll be able to gain some insight to what could be the cause of this issue.

\- Ryan

**Comment by FabulousHotdog** [comment]
ConnorB123 if you can could you please add me on Discord: Fabulous_hotdog#3959 I have a question.

**Comment by rlguy** [comment]
Hello FabulousHotdog,

Are you experiencing this issue also? If your discussion is related to this issue, feel free to provide input here. Any information could be helpful towards solving this issue.

\- Ryan

**Comment by FabulousHotdog** [comment]
It was more with the terms of building the Github code with visual studio.

**Comment by rlguy** [comment]
FabulousHotdog, let's please keep this issue thread on topic. I am sure there are better and more appropriate methods of contacting a GitHub user than through this issue.

**Comment by FabulousHotdog** [comment]
GitHub does not have an option to DM or show contact information.

**Comment by ConnorB123** [comment]
Hi,

Any update with this? If I could use the network render it would help speed up my workflow a lot.

**Comment by rlguy** [comment]
Hello ConnorB123,

I only had a chance to test this very briefly last Monday, but was not able to reproduce this issue. I still want to test this more extensively using a higher resolution simulation. At the moment, development and bug fixes for the FLIP Fluids addon is on hold. I am currently in the process of moving homes and do not have access to my desktop system used for development. Development should resume as normal within the next few days however. Sorry for this inconvenience!

I would like to test this issue using the exact Blender builds that you are running. Would you be able to attach the Blender System Info files for both of your systems. This file can be retrieved through Blender: **Blender > Help > Save System Info**. This file will give me more info on your Blender setup.

\- Ryan

**Comment by Blenderari** [comment]
Hi!
I'm having the same issue.
I'm using Flip Fluid a lot and I'm very very happy with it :)
But I'm having a weird issue with my last animation.
Rendering the animation, sometimes, not always, just sometimes and randomly (If i try to render the same frame the second times it works....) When I rendered a frame, just after few second, during the preparation for the next frame, Blender crash and i close itself.....
If I reopen the scene, it works fine until, randomly, it crash between some other frames rendering.....the crash always happen the end of a frame and the start of the successive....

My pc is composed by

Xeon 2695V3
32gb ram
3x quadro P4000
Windows 7 pro

my blender version is the official 2.79 and the flip fluid version is the last one

The crashes always seem to happen either just after a frame finishes rendering, or right as the next one starts, and the frame is not the issue because if i try to render it again it works and it will crash after some other random frames.

Thanks for any help

**Comment by rlguy** [comment]
I will be closing this issue due to inactivity and to clean up the issue tracker. If there are any further updates, feel free to add them here or send us an email at flip.fluids@gmail.com

**Comment by PascalSchimmler** [comment]
Same Issue here in Blender 2.8 and Flip-Fluid from 14.02.2019. Currently, the only well working workaround is to render from console.

Issue: When trying to render as animation or as image, while initializing blender crashes. It seems to be more likely to work, when rendering a single image.

**Comment by rlguy** [comment]
Hi PascalSchimmler,

Thanks for the report! I have a few questions that could help me understand this issue:
- Are you able to render successfully without using Network Render, or does this only happen when using the Network Render addon?
- Does this happen when rendering with EEVEE, or with Cycles, or both?

\- Ryan

**Comment by Krapisa** [comment]
Hi,
can confirm for me Cycles is crashing upon starting a render, single frame or animation does not mattery, rendering from command line seems to work.

Im not using any Network Render atm and haven`t tried it with Eevee yet

Cheers,

**Comment by gradynelson** [comment]
I'm having this issue in heavy particle animated scenes and this is my fix: if I use the command line for rendering, keep the auto-ram cleaning open and instead of getting all the frames with " -a " in cmd I use a for loop to open separate shell for each frame.

**Comment by rlguy** [comment]
Cleaning up the issue tracker, apologies for the ping.

Closing this issue as the Network Render Addon is no longer available in supported versions of Blender.
